InitBuilder.js

```diff
@@ -0,0 +1,79 @@
+
+/*
+new IntBuilder(int) // constructor takes starting integer, if not passed starts with 0;
+.plus(...n)         // take infinite number of integers and sum all with stored value;
+.minus(...n)        // take infinite number of integers and subtract from stored value; 
+.multiply(n)        // multiply param n on stored value;
+.divide(n)          // leaves integer part of division stored value on n; 
+.mod(n)             // leaves remainder of the division stored value with on n;
+.get()              // returns stored value;
+*/
+"use strict"
+
+class  baseBilder{  // 
+    constructor(value){
+        this.value = value 
+    }
+    plus(...n) {
+        this.value =  this.value  + n.reduce((sum, value) => sum + value) // this
+        return this
+    }
+    divide(n){
```

Comment:
looks like divide function should behaves differently for IntBuilder and StringBuilder - in this case it should not be in the base class